src/navigation/navigation.hpp

```diff
@@ -212,35 +221,25 @@ namespace navigation {
       // Counter of how many IMUs have failed
       uint32_t nOutlierImus_;
 
-      // Stripe counter (rolling values)
-      DataPoint<uint32_t> stripe_counter_;
-      // Keyence data read
-      KeyenceDataArray keyence_readings_;
-      // Previous keyence data for comparison
-      KeyenceDataArray prev_keyence_readings_;
-      // Are the keyence sensors used or ignored?
+      // Stripe counter object
+      StripeCount stripe_counter_;
+      // Flags if keyences are used and if real
       bool keyence_used_;
-      // Is the keyence used fake or real?
       bool keyence_real_;
-      // This counts the number of times the keyence readings disagree with the IMU data more than
-      // allowed due to uncertainty. It is used at the moment to check if the calculated
-      // uncertainty is acceptable.
-      uint32_t keyence_failure_counter_;
-
 
       // To store estimated values
       ImuDataPointArray sensor_readings_;
       DataPoint<NavigationType> acceleration_;
       DataPoint<NavigationType> velocity_;
-      DataPoint<NavigationType> distance_;
+      DataPoint<NavigationType> displacement_;
       NavigationVectorArray gravity_calibration_;
 
       // Initial timestamp (for comparisons)
       uint32_t init_timestamp_;
       // Previous timestamp
       uint32_t prev_timestamp_;
       // Uncertainty in distance
-      NavigationType distance_uncertainty_;
+      NavigationType displ_unc_;
       // Uncertainty in velocity
       NavigationType velocity_uncertainty_;
```

Comment:
Both should be entirely shortened (I changed that), but I don't see it being confusing because where they are defined, it is clearly commented that those are the uncertainties in displacement and velocity.